### ISSUE TITLE
what is richText.type.transformPresence ?

### ISSUE THREAD

**Issue by welkinwong** [share/sharedb#369]
I found this from /examples/rich-text-presence/rich-text.js

```
var richText = require('rich-text');

richText.type.transformPresence = function(presence, op, isOwnOp) {
  if (!presence) {
    return null;
  }

  var start = presence.index;
  var end = presence.index + presence.length;
  var delta = new richText.Delta(op);
  start = delta.transformPosition(start, !isOwnOp);
  end = delta.transformPosition(end, !isOwnOp);

  return Object.assign({}, presence, {
    index: start,
    length: end - start
  });
};

module.exports = richText;
```

but delete it, still works
so do i need this ?

**Comment by alecgibson** [comment]
What still works? The presence updates? It shouldn't.

**Comment by welkinwong** [comment]
> What still works? The presence updates? It shouldn't.

yes, the presence updates,  so why is this used?

**Comment by alecgibson** [comment]
Oh, actually I forgot that this PR was finally accepted and published: https://github.com/ottypes/rich-text/pull/32

This shouldn't be needed any more if we make sure that `rich-text` uses at least `4.1.0`.

**Comment by welkinwong** [comment]
> Oh, actually I forgot that this PR was finally accepted and published: [ottypes/rich-text#32](https://github.com/ottypes/rich-text/pull/32)
> 
> This shouldn't be needed any more if we make sure that `rich-text` uses at least `4.1.0`.

Thanks for the answer! : )
